src/tests/GOTestExe.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2023-2024 GrandOrgue contributors (see AUTHORS)
+ * License GPL-2.0 or later
+ * (https://www.gnu.org/licenses/old-licenses/gpl-2.0.html).
+ */
+
+#include "GOTestCollection.h"
+#include "GOTestOrganModel.h"
+#include "GOTestSwitch.h"
+#include "GOTestWindchest.h"
+#include <iostream>
+#include <wx/log.h>
+
+int main() {
+  /*
+      This is the main function that will collect all tests in the collection,
+      then run the whole bunch.
+
+      TODO: It should displays also the tests results
+  */
+
+  /* Instantiate all the test classes here */
+  new GOTestWindchest;
+  new GOTestOrganModel;
+  new GOTestSwitch;
+  /* end of instanciation */
+  GOTestResultCollection test_result_collection;
+  test_result_collection = GOTestCollection::Instance()->run();
+
+  // Display tests results
+  int run_number_ = 0;
+  std::vector<GOTestResult *> test_results
+    = test_result_collection.get_results();
+  wxLogMessage("==================== TESTS RESULTS ====================");
+  for (auto current = test_results.begin(); current != test_results.end();
+       ++current, ++run_number_) {
+    auto test_result = *current;
+    wxLogMessage("-------------------------------------------------------");
+    wxLogMessage(wxString(test_result->GetMessage()));
```

Comment:
If all the wxLogMessages would be rewritten with std::cout you'll not need any wxString here either.